files/shorewall

```diff
@@ -0,0 +1,5 @@
+SHOREWALL_TCP .*?%{WORD:fwrule}\:%{WORD:fwtarget}\:IN\=%{WORD:fwin} OUT\=\s*MAC\=%{DATA:fwmac} SRC\=%{IPORHOST:fwsrc} DST\=%{IPORHOST:fwdst} .*? PROTO\=%{WORD:fwproto} SPT\=%{INT:fwspt:int} DPT\=%{INT:fwdpt:int} .*?
```

Comment:
@alvarosimon for me fwmac is enough. as it is an arbitrary string we can still filter on it if we want to, but it makes no sense because we will be looking at the TCP/UDP level anyway.